addons/l10n_dk/models/account_move.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models, fields, api
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    currency_rate_at_transaction = fields.Float(
```

Comment:
We probably want to prefix the field by `l10n_dk`. I'm not sure if we have a guideline for this.